docs/src/developer/conventions.md

```diff
@@ -10,7 +10,8 @@ to simplify typing them.
 
 - **Reciprocal-space vectors:** ``k`` for vectors in the Brillouin zone,
   ``G`` for vectors of the reciprocal lattice,
-  ``q`` for general vectors
+  ``q`` for phonon vectors,
+  ``p`` for general vectors.
```

Comment:
ok, but you should mention what the difference is (one potentially goes out of the irreducible BZ mesh, right and is like an extra grid to deal with non-crystal-conserving perturabations)